src/style/layouts/header.css

```diff
@@ -0,0 +1,63 @@
+.header-wrapper {
+    position: absolute;
+    width: 100%;
+}
+
+.header-content {
+    display: flex;
+    justify-content: space-between;
+    padding: 41px 0;
+    display: flex;
+}
+
+.header-logo {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+    cursor: pointer;
+}
+
+.logo {
+    font-size: var(--font-size-xl);
+    font-weight: var(--font-weight-bold);
+    line-height: var(--line-height-lg);
+    color: var(--color-Wild-Sand);
+}
+
+.menu-list {
+    display: flex;
+    gap: 32px;
+}
+
+.menu-link {
+    font-size: var(--font-size-sm);
+    line-height: var(--line-height-sm);
+    font-weight: var(--font-weight-default);
+    color: var(--color-Wild-Sand);
+    transition: 0.5s all ease;
+}
+
+.menu-link:first-child {
+    font-weight: var(--font-weight-bold);
```

Comment:
not bold for the first child. Just bold for any child that is being hovered. Add more class active for hovered child